Fails to build on aarch64 (v0.26.0)

**Describe the bug**
Error when building on aarch64

```text
   Compiling tao v0.26.0
   Compiling muda v0.11.5
error[E0308]: mismatched types
  --> /build/cargo-vendor-dir/tao-0.26.0/src/platform_impl/linux/taskbar.rs:69:81
   |
69 |         let handle = unsafe { unity_lib.unity_launcher_entry_get_for_desktop_id(id.as_ptr()) };
   |                                         --------------------------------------- ^^^^^^^^^^^ expected `*const i8`, found `*const u8`
   |                                         |
   |                                         arguments to this method are incorrect
   |
   = note: expected raw pointer `*const i8`
              found raw pointer `*const u8`
note: method defined here
  --> /build/cargo-vendor-dir/tao-0.26.0/src/platform_impl/linux/taskbar.rs:9:3
   |
9  |   unity_launcher_entry_get_for_desktop_id: unsafe extern "C" fn(id: *const i8) -> *const isize,
   |   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                       -------------

For more information about this error, try `rustc --explain E0308`.
error: could not compile `tao` (lib) due to previous error
warning: build failed, waiting for other jobs to finish...
       Error [tauri_cli_node] failed to build app: failed to build app
task: Failed to run task "build": exit status 1
```


**Platform and Versions (please complete the following information):**

```
 rustc 1.78.0-nightly (a4472498d 2024-02-15)

 - system: `"aarch64-linux"`
 - host os: `Linux 6.1.31, NixOS, 23.05 (Stoat), 23.05.426.afc48694f2a`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.13.3`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

The build IS OK on my intel system (also nixOS, using the same flake).

--- a/.changes/fix-linux-arm64.md
+++ b/.changes/fix-linux-arm64.md
@@ -2,4 +2,4 @@
 tao: 'patch:bug'
 ---
 
-Fixed an issue that caused compile errors for Linux Arm64 targets.
+Fixed compile errors for Linux Arm64 targets.

--- a/.changes/fix-linux-arm64.md
+++ b/.changes/fix-linux-arm64.md
@@ -1,0 +1,5 @@
+---
+tao: 'patch:bug'
+---
+
+Fixed an issue that caused compile errors for Linux Arm64 targets.

--- a/src/platform_impl/linux/taskbar.rs
+++ b/src/platform_impl/linux/taskbar.rs
@@ -1,4 +1,4 @@
-use std::ffi::CString;
+use std::ffi::{c_char, CString};
 
 use dlopen2::wrapper::{Container, WrapperApi};
 
@@ -6,7 +6,7 @@ use crate::window::{ProgressBarState, ProgressState};
 
 #[derive(WrapperApi)]
 struct UnityLib {
-  unity_launcher_entry_get_for_desktop_id: unsafe extern "C" fn(id: *const i8) -> *const isize,
+  unity_launcher_entry_get_for_desktop_id: unsafe extern "C" fn(id: *const c_char) -> *const isize,
   unity_inspector_get_default: unsafe extern "C" fn() -> *const isize,
   unity_inspector_get_unity_running: unsafe extern "C" fn(inspector: *const isize) -> i32,
   unity_launcher_entry_set_progress: unsafe extern "C" fn(entry: *const isize, value: f64) -> i32,

